Examples/ObjectiveC/LocationPrivacyExample.m

```diff
@@ -0,0 +1,51 @@
+#import "LocationPrivacyExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLocationPrivacy = @"LocationPrivacyExample";
+
+@interface LocationPrivacyExample () <MGLMapViewDelegate>
+@end
+
+@implementation LocationPrivacyExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    mapView.delegate = self;
+
+    // Enable heading tracking mode so that the puck will appear.
+    mapView.userTrackingMode = MGLUserTrackingModeFollow;
```

Comment:
can't we use "showsUserLocation" instead?

Reply:
we sure can.

Reply:
`showsUserLocation` doesn't re-center the map's camera, if I recall - so using `userTrackingMode` ensures the map view is centered on the user's location.